modules/nextflow/src/main/groovy/nextflow/extension/CH.groovy

```diff
@@ -30,7 +30,14 @@ class CH {
         return (Session) Global.session
     }
 
-    static private Map<DataflowQueue, DataflowBroadcast> bridges = new HashMap<>(10)
+    static class Topic {
+        List<DataflowWriteChannel> sources = new ArrayList<>(10)
+        DataflowBroadcast target = new DataflowBroadcast()
+    }
+
+    static final private Map<String, Topic> allTopics = new HashMap<>(10)
+
+    static final private Map<DataflowQueue, DataflowBroadcast> bridges = new HashMap<>(10)
```

Comment:
I've been meaning to ask, what is the purpose of this bridges map? My guess is that it is used to enable queue channels to be re-used without the old `into` operator, is that right?

Reply:
If my guess is correct, it would illustrate my point that the topic channel is no more a "broadcast" than queue channels, in the sense that they both use `DataflowBroadcast` primarily to support multiple readers. Meanwhile, the topic channel uses `MixOp` to receive from multiple sources